src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Dispute.java

```diff
@@ -503,7 +503,7 @@ public class Dispute extends Auditable<String> {
 	/**
 	 * Signatory Type. Can be either 'D' for Disputant or 'A' for Agent.
 	 */
-	@Schema(description = "Signatory Type. Can be either 'D' for Disputant or 'A' for Agent.", maxLength = 1, nullable = true)
+	@Schema(description = "Signatory Type. Can be either 'D' for Disputant or 'A' for Agent.", nullable = true)
```

Comment:
Why didn't this work? maxLength of 1.

Reply:
Yeah this bug was interesting. It adds an annotation that specifies the string length for the property in the generated spec model which causes a type conversion error when submitting the signatory type as enum based on the model and .net tries to convert it to string from the request since it needs to check string length condition so it throws the following error:

`System.InvalidCastException: Unable to cast object of type 'TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0.DisputeSignatoryType' to type 'System.String'.`